doc/developers/advanced_installation.rst

```diff
@@ -438,3 +438,54 @@ Before using ICC, you need to set up environment variables::
 Finally, you can build scikit-learn. For example on Linux x86_64::
 
     python setup.py build_ext --compiler=intelem -i build_clib --compiler=intelem
+
+.. _arm64_dev_env:
+
+Building and testing for the ARM64 platform
+-------------------------------------------
```

Comment:
Using `---` would make this a subsection of `Alternative compilers`, should these be `==` instead?